e_os.h

```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright 1995-2018 The OpenSSL Project Authors. All Rights Reserved.
  *
- * Licensed under the OpenSSL license (the "License").  You may not use
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
```

Comment:
Typo here. Should be Licence. Would you mind updating here and all the other similar places in the other PRs?